lib/Backend/TalkIntegration.php

```diff
@@ -26,6 +27,7 @@ public function __construct($tlk, $utils) {
         $this->utils = $utils;
         $this->tlk = $tlk;
         $this->config = \OC::$server->get(\OCP\IConfig::class);
+        $this->participantService = \OC::$server->get(\OCA\Talk\Service\ParticipantService::class);
```

Comment:
Why isn't dependency injection being used?

Reply:
The class is being initialized by BackendUtils and initializing the variable that way avoids the need to change the other file too.

Reply:
Fair